packages/main/src/DateTimePicker.ts

```diff
@@ -181,10 +175,13 @@ class DateTimePicker extends DatePicker {
 	/**
 	 * @override
 	 */
-	onResponsivePopoverAfterClose() {
+	async onResponsivePopoverAfterClose() {
 		super.onResponsivePopoverAfterClose();
 		this._showTimeView = false;
 		this._previewValues = {};
+		const popover = await this.getPicker();
+		const clocks = popover && popover.querySelector("[ui5-time-selection-clocks]") as TimeSelectionClocks;
+		clocks && clocks._switchClock(0);
```

Comment:
Can we do it with a private property that we can pass trough hbs and not searching trogh DOM?

Reply:
Unfortunately no, as this is on popover level, and it doesn't know about TimePicker part as an object, so it must be 'retreived' from the DOM in order to set it's property.

Reply:
After discussion we agreed to remove this part for now as it is additional improvement out of current scope.